lighthouse-core/test/gather/gatherers/dobetterweb/response-compression-test.js

```diff
@@ -7,194 +7,187 @@
 
 /* eslint-env jest */
 
+jest.mock('../../../../gather/driver/network.js', () => ({
+  fetchResponseBodyFromCache: jest.fn(),
```

Comment:
Perhaps use `mockDriverSubmodules` once https://github.com/GoogleChrome/lighthouse/pull/12478 lands

Reply:
just landed it 👍